components/InteroperablePlanets.vue

```diff
@@ -79,11 +77,12 @@
     left -23% // (146 - 100) / 2
     top 0
     margin-top -33% // (146 - 80) / 2
-    background radial-gradient(50% 50% at 50% 50%, rgba(75, 125, 254, 0.26) 20%, rgba(66, 81, 250, 0) 100%)
+    background radial-gradient(50% 50% at 50% 53.5%, #1440c9 28.99%, #121e4d 44.96%, #0d1236 70.17%, #000)
```

Comment:
Hmm, the previous radial-gradient worked. You probably need to tweak this again. @nassdonald 

![image](https://user-images.githubusercontent.com/1021101/88412416-63b3cf80-cd8e-11ea-97e8-d0eef722234f.png)

Reply:
Shit! Don't rush yo' CSS folks. 🤦  I'll fix that now.